content/applications/sales/point_of_sale/overview/epos_ssc.rst

```diff
@@ -62,27 +62,24 @@ for example, `https://192.168.1.25`. Then, accept the self-signed certificate.
 .. note::
    Note that the protocol is now **HTTPS**.
 
-Click on :menuselection:`Not secure --> Certificate is not valid`.
+Click on :menuselection:`Connection is not secure --> Certificate is not valid`.
 
 .. image:: epos_ssc/browser-warning.png
    :align: center
    :alt: The web browser indicates that the connection to the printer is not secure.
 
-Go to the *Details* tab and click on **Export**.
+Go to the *Details* tab and click on **Copy to file**.
+Select X.509 in base 64 and save it.
 
-.. image:: epos_ssc/certificate-details.png
-   :align: center
-   :alt: Details of the ePOS printer certificate
-
-Import the Self-signed certificate to your browser (Chrome)
+Import the Self-signed certificate to windows (Using Chrome)
 ===========================================================
```

Comment:
```suggestion
============================================================
```
Should be the same length as the title (--> `WARNING: Title underline too short` --> red runbot)